docs/snapshotting/snapshot-support.md

```diff
@@ -213,11 +213,11 @@ file created by a `/snapshot/create` API call and the layer as a memory file
 created by a subsequent `/snapshot/create` API call. The order in which the
 snapshots were created matters and they should be merged in the same order
 in which they were created. To merge a `diff` snapshot memory file on
-top of a base, users should copy its content over the base, as the following
-example does:
+top of a base, users should copy its content over the base. This can be done
+using the `rebase-snap` tool provided with the firecracker release:
```

Comment:
I think it's worth adding or mentioning exactly what the program does, either here or in a `--help` on the binary. Something along the lines of "copies all non-sparse sections from diff file onto the base file".

Reply:
Good callout. Added this description to the `--help` on the binary.